server/src/main/java/com/datastrato/gravitino/server/GravitinoServer.java

```diff
@@ -84,12 +85,12 @@ protected void configure() {
     metricsSystem.register(httpServerMetricsSource);
 
     Servlet servlet = new ServletContainer(this);
-    server.addServlet(servlet, "/api/*");
+    server.addServlet(servlet, API + "*");
     Servlet configServlet = new ConfigServlet(serverConfig);
     server.addServlet(configServlet, "/configs");
-    server.addCustomFilters("/api/*");
-    server.addFilter(new VersioningFilter(), "/api/*");
-    server.addSystemFilters("/api/*");
+    server.addCustomFilters(API + "*");
+    server.addFilter(new VersioningFilter(), API + "*");
+    server.addSystemFilters(API + "*");
```

Comment:
Given in all cases you have you need + " *", perhaps just make the API have the value "/api/*"?

Reply:
private static final String API = "/api/"; well its the same.